packages/main/src/themes/sap_horizon_dark/ColorPalette-parameters.css

```diff
@@ -12,10 +12,14 @@
 	--_ui5_color-palette-item-before-focus-offset: -0.3125rem;
 	--_ui5_color-palette-item-before-focus-hover-offset: -0.0625rem;
 	--_ui5_color-palette-item-after-focus-color: 0.0625rem solid var(--sapContent_ContrastFocusColor);
+	--_ui5_color-palette-item-selected-focused-border-after: none;
 	--_ui5_color-palette-item-after-focus-offset: -0.1875rem;
 	--_ui5_color-palette-item-after-focus-hover-offset: 0.0625rem;
 	--_ui5_color-palette-item-before-focus-border-radius: 0.4375rem;
 	--_ui5_color-palette-item-after-focus-border-radius: 0.3125rem;
 	--_ui5_color-palette-item-hover-outer-border-radius: 0.4375rem;
 	--_ui5_color-palette-item-hover-inner-border-radius: 0.375rem;
+	--_ui5_color-palette-item-selected-focused-border-before: -0.1875rem;
+	--_ui5_color-palette-item-after-focus-color-phone: none;
+	--_ui5_color-palette-item-phone-selected-focus: 0.0625rem solid var(--sapGroup_ContentBackground);
```

Comment:
Do we have a focus representation on mobile? The navigation there should be with an additional keyboard by pressing the tab key. The tab kay closed the dialog though.